Mathlib/MeasureTheory/Function/Holder.lean

```diff
@@ -0,0 +1,262 @@
+/-
+Copyright (c) 2025 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.Analysis.Normed.Module.Dual
+import Mathlib.MeasureTheory.Integral.Bochner
+import Mathlib.Order.CompletePartialOrder
```

Comment:
That's a weird import to need here. `CompletePartialOrder` is domain theoretic

Reply:
:fearful: Well that's *very* scary. On this commit 1f6d64fb841c128018fdb277b3e81a6caff6b1d1, running `#min_imports` at the bottom produces this list! And yet I can remove this import with no consequences! And `Analysis.InnerProductSpace.Basic` is also unnecessary! What is going on here?